.github/workflows/dev_labeler/labeler.yml

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 lang-rust:
-  - rust/**/*
+  - rust/**
```

Comment:
I am not sure this will work. At least the [documentation requests](https://github.com/actions/labeler#basic-examples) `**/*`. If you tested and it still works, fine by me.

Reply:
No, I didn't test, I just saw a bunch of failures from this job so figured I'd try to change this. But now I see that this *is* the fix and you just added this yesterday, so I'll revert.